react-app/src/components/Opensearch/main/Filtering/Export/index.tsx

```diff
@@ -16,22 +16,23 @@ import {
   TooltipTrigger,
   useOsUrl,
 } from "@/components";
+import { ConfirmationDialog } from "@/components/ConfirmationDialog";
 
 import { DEFAULT_FILTERS } from "../../useOpensearch";
 
 export const OsExportData: FC<{
   columns: OsTableColumn[];
   disabled?: boolean;
-}> = ({ columns, disabled }) => {
+  count: number;
+}> = ({ columns, disabled, count }) => {
   const [loading, setLoading] = useState(false);
+  const [showAlert, setShowAlert] = useState(false);
   const url = useOsUrl();
 
-  const handleExport = async () => {
-    if (disabled) {
-      return;
-    }
-    setLoading(true);
+  const EXPORT_LIMIT = 10000;
```

Comment:
I feel constants should be declared outside of components, but that's a stupid nit